prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctionsTests.kt

```diff
@@ -730,7 +770,7 @@ class CustomFHIRFunctionsTests {
     }
 
     @Test
-    fun `test convertDateToAge 4 days - default`() {
+    fun `test convertDateToAge 4 days - Add value for both but switched order`() {
```

Comment:
This might explain why you had those helper functions, did not realize you would have to account for parameters in either position. Seems like there's gotta be a better way, but I don't know what that is 🤔